ra/1.1/resource-agent-api.md

```diff
@@ -198,90 +251,103 @@ unsupported action the RA shall return an error as defined below.
+    resource in the unpromoted role.
+
+- `notify`
+
+    If the resource requires special coordination when multiple instances are
+    run simultaneously in the cluster, the resource agent should support this
+    action, which should perform such coordination.
+
+    When the RA supports this action, RMs should call the action for all active
+    instances of this particular resource in the cluster before and after any
+    demote, promote, start, or stop action performed on any instance of it.
+
+    How the RM passes useful information to the RA when performing this action
+    is currently left to the RM and RA, but may be formalized in a future
+    version of this standard.
+
+- `promote`
+
+    If the resource agent supports roles, this action must put an active
+    resource in the promoted role.
+
+- `recover`
 
     A special case of the `start` action, this should try to recover a resource
-    locally. 
+    locally.
 
     It is recommended that this action is not advertised unless it is
-    advantageous to use when compared to a stop/start operation.
+    advantageous to use when compared to a stop and start action sequence.
 
-    If this is not supported, it may be mapped to a stop/start action by
-    the RM.
+    If this is not supported, it may be mapped to a stop and start action
+    sequence by the RM.
 
     An example includes "recovering" an IP address by moving it to another
     interface; this is much less costly than initiating a full resource group
-    fail over to another node.
+    fail-over to another node.
 
 - `reload`
 
-    Optional.
-
-    Notifies the resource instance of a configuration change external to
-    the instance parameters; it should reload the configuration of the
+    This should notify the resource instance of a configuration change external
+    to the instance parameters. It should reload the configuration of the
     resource instance without disrupting the service.
 
     It is recommended that this action is not advertised unless it is
-    advantageous to use when compared to a stop/start operation.
-
-    If this is not supported, it may be mapped to a stop/start action by
-    the RM.
+    advantageous to use when compared to a stop and start action sequence.
 
-- `meta-data`
+    If this is not supported, it may be mapped to a stop and start action
+    sequence by the RM.
 
-    Mandatory.
+- `reload-params`
 
-    Returns the resource agent meta data via stdout.
+   This should make effective any changes in instance parameters that have been
+   marked as reloadable as described in **Resource Agent Meta-Data**.
 
 - `validate-all`
 
-    Optional.
-
-    Validate the instance parameters provided.
-
-    Perform a syntax check and if possible, a semantic check on the
-    instance parameters.
+    This should validate the instance parameters provided. The thoroughness of
+    the check may optionally be influenced by **Check Levels**.
 
 
 ### Parameter passing
```

Comment:
I noticed we're making capitalization changes to some headers (e.g., s/Exit status codes/Exit Status Codes/) but not others (e.g., Parameter passing). Highlighting here but it depends on your preference.

Reply:
Good point. It was inconsistent before, so whenever I added or changed a header, I made it consistent, but I didn't update all existing headers. I'm tempted to ignore it for this release, but I can do it if anyone wants.

Reply:
Shrug, I wouldn't blame ya.

Reply:
Fixed with new "Capitalize headings consistently" commit